tests/editor_utils_test.py

```diff
@@ -11,13 +11,18 @@
 
 import json
 import uuid
-import unittest
+from superdesk.tests import TestCase
 import superdesk.editor_utils as editor_utils
 
 from superdesk.editor_utils import Editor3Content
 
 
-class Editor3TestCase(unittest.TestCase):
+class Editor3TestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        if "EMBED_PRE_PROCESS" in self.app.config:
+            del self.app.config["EMBED_PRE_PROCESS"]
```

Comment:
why do we need this?

Reply:
Because the config is global to the whole test, and when I modify `EMBED_PRE_PROCESS` at https://github.com/superdesk/superdesk-core/pull/1807/files#diff-80f1d1a029ab2f40ac5996f23c54282aR1614 it will impact all following test, which may fail if they use embed. I'm not sure it there is a cleaner way.

Reply:
Got it thanks.
I think `setUp` is called before every test anyway and by default `EMBED_PRE_PROCESS` is not set anyway.

Reply:
ahh, ok got it now. You can ignore my previous comment :)